src/design-pages/passwordless_gdm.rst

```diff
@@ -0,0 +1,367 @@
+Passkey authentication Kerberos integration
```

Comment:
Should the design document title be updated?

Reply:
yes, I had done this change locally but I didn't want to push the changes yet. Once I do I'll close this issue.

Reply:
Done